source/includes/_leads_IN_V4.md

```diff
@@ -92,6 +92,9 @@ You will have to replace `{access_token}` with access token you get from [SquadV
 
 `400 Bad Request` if the request payload had some error. Specific error is provided in the response.
 
+`429 Too Many Requests` If number of requests hit is greater than specified limit. Specific error is provided in the response. Default rate limit is 100/minute.
```

Comment:
@kunal5 would be better to specify default rate limit at a single place instead of each api, because right now it might be misconstrued as 100/min for each individual api whereas the limit is per business.

Best to create a new section for rate limiting (same level as "leads" & "errors") and then we can link that section from each api.